capstone-frontend/src/components/CreateClub.jsx

```diff
@@ -17,10 +17,11 @@ class CreateClub extends Component {
     }
     data.ownerID = window.localStorage.getItem("userID");
 
+    console.log("Create club", data)
+
     fetch("/api/clubs", {method: "post", body: JSON.stringify(data)})
-        .then(function() {
-          history.push(`/clubpage/${data.id}`);
-        })
+        .then(resp => resp.json())
+        .then(club => history.push(`/clubpage/${data.id}`))
```

Comment:
Should this read `/clubpage${club.id}` instead of `/clubpage${data.id}`? I believe this was breaking before because data does not have an id, but the response does (since we let Firebase generate our ids, and place them in the response).

Reply:
Did you mean `/clubpage/${club.id}`?

Reply:
I did! Thanks for catching that. The URL should read the exact same as before, but replace data with club.

Reply:
Done.